api/core/app/task_pipeline/workflow_cycle_manage.py

```diff
@@ -266,6 +266,7 @@ def _handle_workflow_run_failed(
             workflow_node_execution.error = error
             workflow_node_execution.finished_at = now
             workflow_node_execution.elapsed_time = (now - workflow_node_execution.created_at).total_seconds()
+            session.add(workflow_node_execution)
```

Comment:
This `workflow_node_execution` got via `_get_workflow_node_execution` should already in the session and don't need to add again.